### PR TITLE
Us002 UI tests

### PR DESCRIPTION
**Pull request by philipwilson5**

### CODE REVIEW

File: .github/workflows/node.js.yml
Comment (AlexMcC02) [inline .github/workflows/node.js.yml:1]: cache: "npm" should be cache: 'npm'

File: app.ts
Comment (AlexMcC02) [inline app.ts:1]: Why remove export?

**Review by AlexMcC02 — CHANGES_REQUESTED** [review]
Address the changes and we're good to go.

**Review by AlexMcC02 — APPROVED** [review]
Looks good!

**Review by ConnMeehan — APPROVED** [review]

**Pull request merged**